desc/compute/utils.py

```diff
@@ -1325,6 +1326,190 @@ def body(i, mins):
     return grid.expand(mins, surface_label)
 
 
+def bounce_integral(eq, lambdas, rho=None, alpha=None, resolution=20):
+    """Returns a method to compute the bounce integral of any quantity.
+
+    The bounce integral is defined as F_ℓ(λ) = ∫ f(ℓ) / √(1 − λ |B|) dℓ, where
+        dℓ parameterizes the distance along the field line,
+        λ is a constant proportional to the magnetic moment over energy,
+        |B| is the norm of the magnetic field,
+        f(ℓ) is the quantity to integrate along the field line,
+        and the endpoints of the integration are at the bounce points.
+    For a particle with fixed λ, bounce points are defined to be the location
+    on the field line such that the particle's velocity parallel to the
+    magnetic field is zero, i.e. λ |B| = 1.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium on which the bounce integral is defined.
+    lambdas : ndarray
```

Comment:
I feel like we should change lambda to lambda_pitch or something else to avoid confusion with the lambda (difference between theta_PEST and theta) used by DESC.